packages/fluentui/react-northstar/src/themes/teams/components/Pill/pillStyles.ts

```diff
@@ -0,0 +1,80 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { PillStylesProps } from '../../../../components/Pill/Pill';
+import { PillVariables } from './pillVariables';
+import { getBorderFocusStyles } from '../../getBorderFocusStyles';
+import { pxToRem } from '../../../../utils/index';
+
+export const pillStyles: ComponentSlotStylesPrepared<PillStylesProps, PillVariables> = {
+  root: ({ props: p, variables: v, theme: { siteVariables } }): ICSSInJSStyle => {
+    return {
+      display: 'inline-flex',
+      width: 'fit-content',
+      height: v.height,
+      maxHeight: v.height,
+      borderRadius: v.borderRadius,
+      background: v.background,
+      margin: v.margin,
+      minWidth: v.minWidth,
+
+      ':hover': {
+        background: v.backgroundHover,
+      },
+
+      ...(p.rounded && {
+        borderRadius: v.roundedBorderRadius,
+        ...((p.size === 'small' || p.size === 'smaller') && {
+          borderRadius: v.smallerRoundedBorderRadius,
+        }),
+      }),
+
+      ...(p.size === 'smaller' && {
+        minWidth: v.smallerMinWidth,
+        margin: v.smallerMargin,
+        height: v.smallerHeight,
+        maxHeight: v.smallerHeight,
+      }),
+
+      ...(p.size === 'small' && {
+        minWidth: v.smallMinWidth,
+        margin: v.smallMargin,
+        height: v.smallHeight,
+        maxHeight: v.smallHeight,
+      }),
+
+      ...(p.disabled && {
+        pointerEvents: 'none',
+        cursor: 'not-allowed',
+        background: v.disabledBackground,
+        color: v.disabledColor,
+        ':hover': {
+          background: v.disabledBackground,
+        },
+      }),
+
+      ...(p.appearance === 'outline' && {
+        borderWidth: pxToRem(1),
```

Comment:
```suggestion
        borderWidth: "1px",
```

Let's use `1px` to avoid rounding from `pxToRem()` (0.0714rem) as it can cause issues.